addons/website_event_booth/controllers/event_booth.py

```diff
@@ -12,18 +12,19 @@
 
 class WebsiteEventBoothController(WebsiteEventController):
 
-    @http.route('/event/<model("event.event"):event>/booth', type='http', auth='public', website=True, sitemap=False)
-    def event_booth_main(self, event, booth_category_id=False, booth_ids=False):
+    @http.route('/event/<model("event.event"):event>/booth/<path:page>', type='http', auth='public', website=True, sitemap=False)
+    def event_booth_main(self, event, page, booth_category_id=False, booth_ids=False):
         try:
             event.check_access_rights('read')
             event.check_access_rule('read')
         except exceptions.AccessError:
             raise Forbidden()
 
+        seo_object = request.website.get_template('website_event_booth.' + page)
```

Comment:
`seo_object` is a very generic name can we name it related to the module something

Reply:
Yup its a generic name but I did it to make the code look consistent :sweat_smile:. 
Please let me know your thoughts on it.